apps/charterafrica/src/payload/blocks/FlourishCharts.js

```diff
@@ -0,0 +1,29 @@
+const FlourishCharts = {
+  slug: "flourish-charts",
+  labels: {
+    singular: {
+      en: "Flourish Charts",
+      fr: "Graphiques Flourish",
+      pt: "Gráficos Flourish",
+    },
+    plural: {
+      en: "Flourish Charts",
+      fr: "Graphiques Flourish",
+      pt: "Gráficos Flourish",
+    },
+  },
+  fields: [
+    {
+      name: "code",
+      label: {
+        en: "Embed Code",
+        fr: "Code d'intégration",
+        pt: "Código de incorporação",
+      },
+      type: "textarea",
```

Comment:
I think it should be [`code`](https://payloadcms.com/docs/fields/code) with `html` as language